beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -279,8 +283,31 @@ func (s *Store) setSeenAtt(a *ethpb.Attestation) error {
 	return nil
 }
 
-// returns true when time is divisible with slot duration / 2.
-func halfSlot(genesisTime uint64) bool {
-	t := time.Unix(int64(genesisTime), 0)
-	return uint64(roughtime.Since(t).Seconds())%params.BeaconConfig().SecondsPerSlot/2 == 0
+// savesNewAttestation saves the new attestations to DB.
+func (s *Store) saveNewAttestation(ctx context.Context, att *ethpb.Attestation) error {
+	r, err := ssz.HashTreeRoot(att)
```

Comment:
This needs to be hash tree root of att.Data

Reply:
Ahh! thank you